types/marshal.go

```diff
@@ -120,6 +120,14 @@ func (txs Txs) Len() int           { return len(txs) }
 func (txs Txs) Less(i, j int) bool { return txs[i].Date > txs[j].Date }
 func (txs Txs) Swap(i, j int)      { txs[i], txs[j] = txs[j], txs[i] }
 
+// make sure not return null
+func (r *TxPage) MarshalJSON() ([]byte, error) {
```

Comment:
do we need to have this change on this side? 

Since non optional `docs` only needed on the response side, api handler should be responsible to construct this

Reply:
removed, now endpoint api needs to call `NewTxPage` or build it by itself